ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,224 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  ashenEyeDirections?: string[];
+}
+
+const triggerSet: TriggerSet<Data> = {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  zoneId: ZoneId.AsphodelosTheThirdCircle,
+  timelineFile: 'p3n.txt',
+  triggers: [
+    {
+      id: 'P3N Experimental Fireplume 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'P3N Experimental Fireplume 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.outOfMiddle!();
+      },
+      outputStrings: {
+        outOfMiddle: {
+          en: 'Out Of Middle',
+          de: 'Raus aus der Mitte',
+          ja: '横へ',
+          cn: '远离中间',
+          ko: '가운데 피하기',
+        },
+      },
+    },
+    {
+      id: 'P3N Scorched Exaltation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B8', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Heat of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B2', source: 'Phoinix' }),
+      suppressSeconds: 1,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'P3N Darkened Fire Aoe',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: ['010[C-F]'] }),
```

Comment:
```suggestion
      netRegex: NetRegexes.headMarker({ id: '010[C-F]' }),
```

Reply:
Changed in 1be9b4e868f8ab0e3c7278e054e1fb9fb7df5be7